config/routes.rb

```diff
@@ -162,64 +164,64 @@
+  get  "heartables/:heartable_id/lovers", controller: :heartables, action: :lovers, as: :heartables_lovers
 
   resources :stories, only: [:show]
 
-  get '/leaderboards' => 'leaderboards#index'
+  get "/leaderboards", to: "leaderboards#index"
 
   # Webhooks
   namespace :webhooks do
-    post '/assembly_assets/transaction' => 'assembly_assets#transaction'
-    post '/mailgun' => 'mailgun#create'
-    post '/mailgun/reply' => 'mailgun#reply'
-    post '/github' => 'github#create'
-    post '/landline' => 'landline#create'
-    post '/readraptor/immediate/:entity_id' => 'read_raptor#immediate', as: :readraptor_immediate
-    post '/readraptor/daily'          => 'read_raptor#daily'
-    post '/readraptor/unread_comment' => 'read_raptor#unread_coment', as: :readraptor_unread_comment
-    post '/pusher' => 'pusher#auth'
+    post "/assembly_assets/transaction", to: "assembly_assets#transaction"
+    post "/mailgun", to: "mailgun#create"
+    post "/mailgun/reply", to: "mailgun#reply"
+    post "/github", to: "github#create"
+    post "/landline", to: "landline#create"
+    post "/readraptor/immediate/:entity_id", to: "read_raptor#immediate", as: :readraptor_immediate
+    post "/readraptor/daily",          to: "read_raptor#daily"
+    post "/readraptor/unread_comment", to: "read_raptor#unread_coment", as: :readraptor_unread_comment
+    post "/pusher", to: "pusher#auth"
   end
 
   # Facebook
-  get '/channel.html' => 'facebook#channel', :as => :facebook_channel
+  get "/channel.html", to: "facebook#channel", as: :facebook_channel
 
   # Exceptions
-  get "/404", :to => "errors#not_found"
-  get "/500", :to => "errors#error"
-  get "/errors/crash", :to => "errors#test_exception"
-  get "/errors/maintenance", :to => "errors#maintenance"
-  get "/errors/heroku", :to => "errors#test_heroku"
+  get "/404", to: "errors#not_found"
+  get "/500", to: "errors#error"
+  get "/errors/crash", to: "errors#test_exception"
+  get "/errors/maintenance", to: "errors#maintenance"
+  get "/errors/heroku", to: "errors#test_heroku"
 
   resources :tags, only: [] do
-    post 'follow'
-    post 'unfollow'
+    post "follow"
+    post "unfollow"
   end
 
   # Help
-  get '/help/:group', :to => 'questions#index', :as => :help
-  get '/help' => redirect('/help/basics'), :as => :faq
+  get "/help/:group", to: "guides#faq", as: :help
+  get "/help", to: redirect("/help/basics"), as: :faq
 
-  get '/metrics' => 'metrics#overview'
+  get "/metrics", to: "metrics#overview"
 
   # Guides
-  get '/guides/:group', :to => 'guides#index', :as => :guides
-  get '/guides', :to => 'guides#index'
+  get "/guides/:group", to: "guides#guides", as: :guides
+  get "/guides", to: "guides#guides"
 
   # redirect support-foo to helpful
-  get '/support-foo', to: redirect('/helpful')
-  get '/support-foo/*extra', to: redirect {|p, req| "/helpful/#{p[:extra]}" }
+  get "/support-foo", to: redirect("/helpful")
+  get "/support-foo/*extra", to: redirect {|p, req| "/helpful/#{p[:extra]}" }
```

Comment:
Unused block argument - `req`. If it's necessary, use `_` or `_req` as an argument name to indicate that it won't be used.<br>Space between { and | missing.